src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+    private static final String FORMAT = "%02x";
+
+    public HashUtil() {
```

Comment:
in utility classes, constructors should be private, to prevent instantiation (it's not needed, there are only static methods). I'm sorry it wasn't pointed out before! 😿